app/views/content_items/service_manual_topic.html.erb

```diff
@@ -22,7 +22,11 @@
       <%= @content_item.description %>
     </p>
 
-    <div class="js-hidden" data-module="accordion-with-descriptions">
+    <% if @content_item.visually_collapsed? %>
```

Comment:
Yeah.. I nearly went for a `<%= content_tag :div, some_hash_I've_sculpted %>` but it still needed me to define a hash with a condition that looked like.. `{class: "js-hidden", data: { module: "accordion-with-descriptions" } }`. It was less readable :( I didn't really want to push it into a helper either as it would hide what it's up to.. sigh.

The accordion is the thing that does the collapsing.. If it's expanded then we don't want the accordion.